packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -804,9 +808,27 @@ export default class Meetings extends WebexPlugin {
       );
 
       this.meetingCollection.set(meeting);
+      const isMeetingActive = destination.fullState.active;
+      const syncMeetingsIsSource = (source === 'syncMeetings');
```

Comment:
whats the intention of source= syncMeeting here

Reply:
it is to avoid running the random timer if the meeting bricklet appearance is triggered by syncMeetings (which is when the user loads the page). We just want to add the timer to those meetings that appear as a product of handleLocusMercury (meetings starting in the next 5 mins) when the page is already loaded.
I have change this so that handleLocusMercury sends source='handleLocusMercury'), to make sure only in the above case, the timer runs.

Reply:
Should we change this to boolean rather then the name 'handleLocusMercury'

@marcin-bazyl